python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -708,20 +763,32 @@ def flush(self, ids):
                 if this_id not in self._fully_loaded.keys():
                     self._fully_loaded[this_id] = self.objects[this_id]
 
+                subobj_attr = getattr(self.objects[this_id], self.sub_split, None)
+                sub_attr_dirty = getattr(subobj_attr, '_dirty', False)
```

Comment:
All `GangaObject`s will have the `_dirty` flag so do we need to check for its existence here?